docs/basic_training/index.md

```diff
@@ -20,19 +20,20 @@ By the end of this course you should:
 
 ## Follow the training videos
 
-In our latest [nf-core training in March, 2023](https://nf-co.re/events/2023/training-march-2023), we used this training material to teach Nextflow and the recording is available for you to watch at your pace in multiple languages.
+In our latest [nf-core training in September, 2023](https://nf-co.re/events/2023/training-basic-2023), we used this training material to teach Nextflow and the recording is available for you to watch at your pace below.
 
-Check the links below for the YouTube videos in the available languages:
+<div style="text-align: center;">
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/watch?v=ERbTqLtAkps&list=PL3xpfTVZLcNiLFLiDqk_H5b3TBwvgO_-W" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen="" data-ruffle-polyfilled=""></iframe>
+</div>
+
+We also recorded the [nf-core training in March, 2023](https://nf-co.re/events/2023/training-march-2023) in multiple languages. You can access them below.
```

Comment:
```suggestion
If English is not your preferred language, you may find it useful to follow the training from the [March 2023 event](https://nf-co.re/events/2023/training-march-2023) , which we ran in multiple languages.
Please note that some parts of the training material may have been updated since this was recorded.
```